File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio11.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio11 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o valor de a: ");
        int a = entrada.nextInt();
        System.out.println("Insira o valor de b: ");
        int b = entrada.nextInt();
        System.out.println("Insira o valor de c: ");
        int c = entrada.nextInt();

        int delta = (b*b) - 4 * a * c;
        System.out.println("O valor de delta é: " +delta);
    }
}

File: guanabara-c/lista01/exercicio11.c
#include <stdio.h>

int main(){
    int a, b, c, delta;

    printf("Insira o valor de a: ");
    scanf("%d", &a);
    printf("Insira o valor de b: ");
    scanf("%d", &b);
    printf("Insira o valor de c: ");
    scanf("%d", &c);
    delta = (b * b) - 4 * a * c;
    printf("O valor de delta é: %d", delta);

    return 1;
}
